index.html

```diff
@@ -269,6 +269,51 @@ <h4>Skills</h4>
               </div>
               </div>
         </section>
+        
         </div>
+        <section class="section">
+            <div class="contact-text-container">
+             <h3 class="contact-text">I'm always interested in hearing about new projects so if you'd like to chat please get in</h3>
+            <form method="post" action="https://formspree.io/f/mrgjwppa">
+            <input class="input" required maxlength="30" name="name" type="text" placeholder="Full Name" value="" />
+            <input class="input" required type="email" placeholder="Email Address" value=""/>
+            <textarea maxlength="500" name="text" required class="area"> Enter text here </textarea>
```

Comment:
- [Optional] Kindly add the attribute placeholder for adding placeholder text , avoid putting it as plain text between tags .

**Instead of**  
`<textarea maxlength="500" name="text" required class="area"> Enter text here </textarea>`

**You can do**
`<textarea maxlength="500" name="text" required class="area" placeholder="Enter text here"></textarea>`